src/main/conversions/v2.21.0/c2_21_0_2018050101.clj

```diff
@@ -0,0 +1,44 @@
+(ns facepalm.c2-21-0-2018050101
+  (:use [kameleon.sql-reader :only [load-sql-file exec-sql-statement]]))
+
+(def ^:private version
+  "The destination database version."
+  "2.21.0:20180501.01")
+
+(defn- add-uniqueness-constraint
+  []
+  (exec-sql-statement
+   "ALTER TABLE ONLY interactive_apps_proxy_settings
+    ADD CONSTRAINT interactive_apps_proxy_settings_id_unique UNIQUE (id)"))
+
+(defn- drop-constraint
+  []
+  (exec-sql-statement
+   "ALTER TABLE ONLY interactive_apps_proxy_settings
+    DROP CONSTRAINT IF EXISTS interactive_apps_proxy_settings_container_settings_id_fkey"))
+
+(defn- remove-column
+  []
+  (exec-sql-statement
+   "ALTER TABLE ONLY interactive_apps_proxy_settings
+    DROP COLUMN IF EXISTS container_settings_id"))
```

Comment:
Since `tables/84_interapps_proxy_settings.sql` (added by #23) isn't in prod yet, we don't need to alter it in this conversion; because we'll import prod's data (without this table), and the `2.21.0:20180426.01` conversion will add this table with the new column already in `tables/84_interapps_proxy_settings.sql` (added by this PR).

Same for the dropped constraint in `constraints/84_interapps_proxy_settings.sql` (it doesn't need to be removed in this conversion).

Reply:
For that matter, we don't even need a new conversion either. The new `container_settings` column and constraint can be safely added to the previous conversion.

Reply:
My feeling on this is that it doesn't hurt to have this conversion act as though the intent of the previous conversion happened. Attempting to drop the columns with these statements won't do any harm, and it does ensure that this conversion will succeed if it's ever applied to a database that was converted to version `2.21.0:201804226.01`.